core/src/main/java/org/testcontainers/utility/CommandLine.java

```diff
@@ -57,15 +59,19 @@ public static String runShellCommand(String... command) {
      */
     public static boolean executableExists(String executable) {
 
+        String winExecutable = executable + ".exe";
+
         // First check if we've been given the full path already
         File directFile = new File(executable);
-        if (directFile.exists() && directFile.canExecute()) {
+        File windowsExeFile = new File(winExecutable);
```

Comment:
Thank you for a review!
On Windows there is no chance to run an executable without ".exe" extension (at least without some dirty hacks what is out of scope) so your first advice is very good. We never need to check a filename without extension on Windows.
So I've made the advised change.